src/api/app.js

```diff
@@ -1,6 +1,6 @@
 const safeApp = require('@maidsafe/safe-node-app');
 const ipc = require('./ipc');
-const { genHandle, getObj, freeObj, netStateCallbackHelper } = require('./helpers');
+const { genHandle, getObj, netStateCallbackHelper } = require('./helpers');
 
 module.exports.manifest = {
```

Comment:
We are missing some new functions that are now exposed from safe_app_nodejs: `app.clearObjectCache`, `app.isMockBuild`, `getAccountInfo`, `isNetStateInit`, `isNetStateConnected` and `isNetStateDisconnected`.
We'd also have to exposed the constants, but that's a bit more complex and we can do it in a separate PR.